android/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM gitpod/workspace-full
+
+USER root
+ARG SDK_VERSION=commandlinetools-linux-7302050_latest.zip
+ARG ANDROID_BUILD_VERSION=30
+ARG ANDROID_TOOLS_VERSION=30.0.3
+ARG NDK_VERSION=21.4.7075529
+
+# set default environment variables, please don't remove old env for compatibilty issue
+ENV ADB_INSTALL_TIMEOUT=10
+ENV ANDROID_HOME=/opt/android
+ENV ANDROID_SDK_HOME=${ANDROID_HOME}
+ENV ANDROID_SDK_ROOT=${ANDROID_HOME}
+ENV ANDROID_NDK=${ANDROID_HOME}/ndk/$NDK_VERSION
+ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+
+ENV PATH=${ANDROID_NDK}:${ANDROID_HOME}/cmdline-tools/latest/bin:${ANDROID_HOME}/emulator:${ANDROID_HOME}/platform-tools:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${PATH}
+
+# Android now only runs on jdk-8, jdk-11 is on the list
+RUN apt update -qq && apt install -qq -y --no-install-recommends \
+        ruby-dev \
+        openjdk-8-jdk-headless \
+        openjdk-11-jdk-headless \
+    && gem install bundler \
+    && rm -rf /var/lib/apt/lists/*;
```

Comment:
Possibly rely on `install-packages` script provided on `workspace-full`, since it'll do the `apt update` and `apt install -y --no-install-recommends` for you. It'll also

```dockerfile
# Skip installing ruby-dev package from Ubuntu since workspace images has Ruby Version Manager.
# Also, this will trigger DL3004 (https://github.com/hadolint/hadolint/wiki/DL3004) when linted against Hadolint,
# so please ignore that.
RUN sudo install-packages openjdk-8-jdk-headless openjdk-11-jdk-headless \
        # Just upgrade bundler, but ping the maintainers regarding upgrading Ruby within RVM via an merge request.
        && gem install bundler
```